onnxruntime/core/flatbuffers/schema/ort.py.fbs

```diff
@@ -0,0 +1,245 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace ort_flatbuffers_py.experimental.fbs;
+
```

Comment:
I didn't check this in previously as I considered it to be a temporary hack to generated the python bindings with a usable namespace. Checking it in might also be confusing - e.g. should you edit this and the ort.fbs file, and how are the namespaces for each handled.

Is there a need or reason to check it in now?

Reply:
Added a .gitignore here to prevent this to happen in the future